src/robot.coffee

```diff
@@ -217,6 +217,24 @@ class Robot
       ((msg) -> msg.message = msg.message.message; callback msg)
     )
 
+  # Public: Adds a Listener that triggers when no other text matchers match
+  # and the message was addressing the robot.
+  #
+  # callback - A Function that is called with a Response object.
+  #
+  # Returns nothing.
+  catchAllAddressed: (callback) ->
```

Comment:
Doh! Forgot to include `options`. Thinking of tests to add to prevent that mistake in the future...